homeassistant/components/daikin/sensor.py

```diff
@@ -179,11 +214,18 @@ def __init__(
         """Initialize the sensor."""
         self.entity_description = description
         self._api = api
-        self._attr_name = f"{api.name} {description.name}"
+        self._attr_entity_registry_enabled_default = description.enabled_default
+        if description.cross_device:
+            self._attr_name = f"Daikin {description.name}"
+        else:
+            self._attr_name = f"{api.name} {description.name}"
 
     @property
     def unique_id(self) -> str:
         """Return a unique ID."""
+        if self.entity_description.cross_device:
+            # Cross-device sensors should be generated once for all devices
+            return f"daikin-{self.entity_description.key}"
```

Comment:
The entity registry is aware of both the integration domain and the platform domain. We don't need to prefix the unique id with the integration domain.